x/oracle/keeper/upgrade.go

```diff
@@ -39,21 +39,35 @@ func (k Keeper) GetOracleUpgradeInfo(ctx sdk.Context) (*types.OracleUpgradeInfo,
 	return &upgradeInfo, nil
 }
 
-func (k Keeper) RemoveOracleUpgradeInfo(ctx sdk.Context) {
-	store := ctx.KVStore(k.storeKey)
-
-	store.Delete(types.OracleUpgradeInfoKey)
-}
-
 func (k Keeper) ApplyUpgrade(ctx sdk.Context, info *types.OracleUpgradeInfo) error {
+
 	params := k.GetParams(ctx)
 	params.UniqueId = info.UniqueId
 	if err := params.Validate(); err != nil {
 		return err
 	}
 	k.SetParams(ctx, params)
 
-	//TODO: update `Oracles` that already upgraded
+	iterator := k.GetOracleUpgradeIterator(ctx, info.UniqueId)
+	defer iterator.Close()
+
+	for ; iterator.Valid(); iterator.Next() {
+		uniqueID, address := types.SplitOracleUpgradeKey(iterator.Key())
+		oracleUpgrade, err := k.GetOracleUpgrade(ctx, uniqueID, address.String())
+		if err != nil {
+			return err
+		}
```

Comment:
I think you can change like below:
We don't have to called GetOracle(...) function.
```suggestion
		bz := iterator.Value()
		oracleUpgrade := &types.OracleUpgrade{}

		if err := k.cdc.UnmarshalLengthPrefixed(bz, oracleUpgrade); err != nil {
			return err
		}
```

Reply:
I fixed it in [0e5ad56](https://github.com/medibloc/panacea-core/pull/586/commits/0e5ad56b39f93bb576e7249e4d4d0eb114687c53). Please check again, thanks.